.alexrc

```diff
@@ -1,3 +1,3 @@
 {
-  "allow": ["hook", "hooks", "color", "colors", "husky", "american"]
+  "allow": ["hook", "hooks", "color", "colors", "husky", "american", "whitelist"]
```

Comment:
Let's not include "whitelist". 

We're using the nomenclature "allowlist" or "denylist" to avoid using color as a way to express validity.

Reply:
I understand the concern. However, `audit-ci`'s interface uses the word `whitelist`, so this is just to document that accordingly.

As a maintainer of `audit-ci`, I will strongly consider supporting the suggested nomenclature in the future.